controller/src/main/java/gov/va/vro/controller/exception/GlobalExceptionHandler.java

```diff
@@ -141,14 +141,16 @@ public ResponseEntity<ClaimProcessingError> handleException(Exception exception)
     return new ResponseEntity<>(cpe, HttpStatus.INTERNAL_SERVER_ERROR);
   }
 
+  /**
+   * Handles BIP Exceptions.
+   *
+   * @param exception the exception
+   * @return returns response
+   */
   @ExceptionHandler(BipException.class)
   public ResponseEntity<ClaimProcessingError> handleBipClaimException(BipException exception) {
     log.error("BIP exception: {}", exception.getMessage(), exception);
     ClaimProcessingError cpe = new ClaimProcessingError(exception.getMessage());
-    return new ResponseEntity<>(
```

Comment:
BIP does return 404 (NOT_FOUND) when it cannot find the claim for the given claim ID. It throws the exception for claim not found ONLY in some cases. If you remove this part of the code, it will make automatedClaim return 404 instead of 400 for those BIP return 404.

Reply:
That is OK. It is worse than now which always throws 500.

Reply:
This part has nothing to do with 500. It only changes 404 to 400 as you requested before. But it is really that important anyway.